components/ai/code/ai-bbe/create-images-with-dall-e.md

```diff
@@ -1,4 +1,4 @@
 ---
 title: 'Create images with DALL-E and Stable Diffusion'
-description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. Ballerina’s powerful standard library makes uploading, downloading and processing images a breeze."
+description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. The powerful Ballerina libray makes uploading, downloading and processing images a breeze."
```

Comment:
WDYT?
```suggestion
description: "Stable Diffusion and OpenAI’s DALL-E image APIs generate or edit images with text based instructions. The power of Ballerina libraries makes uploading, downloading and processing images a breeze."
```

Reply:
Saying `ballerina libraries` is incorrect as per https://github.com/ballerina-platform/ballerina-dev-website/pull/7782#issuecomment-1723704068. Instead, we can have the sentence below.

`The power of the Ballerina library makes uploading, downloading, and processing images a breeze.`

@anupama-pathirage

Reply:
+1

Reply:
Updated.